FortnoxAPILibrary/Connectors/InvoiceConnector.cs

```diff
@@ -299,7 +309,19 @@ public bool NotCompleted
 			}
 		}
 
-		private bool filterBySet = false;
+        /// <summary>
+        /// Use with Find() to limit the search result
+        /// </summary>
+        [FilterProperty("credit")]
+        public bool IsCredit { get; set; }
+
+        /// <summary>
+        /// Use with Find() to limit the search result
+        /// </summary>
+        [FilterProperty("invoicetype")]
+        public InvoiceType InvoiceTypeFilter { get; set; }
```

Comment:
`InvoiceTypeFilter` => `InvoiceType` also `[FilterProperty("invoicetype")]` => `[FilterProperty]`
1. This is not a filter, it's a search parameter (check documentation). I think also FilterProperty is wrongly named, it should be named SearchProperty, but it's out of this scope.
2. Why add suffix when no other search parameter have this?

Reply:
2. Done this way because of InvoiceType enum being declared inside this class, so can not use InvoiceType name for the property.

1. InvoiceTypeSearchParameter?

Reply:
Argh... Yeah, it's a problem, this SDK is so weirdly implemented. I think keep InvoiceTypeFilter in that case, we'll change it if we fix FilterProperty sometime.